golang/cosmos/app/app.go

```diff
@@ -577,14 +580,28 @@ func NewAgoricApp(
 	icaHostIBCModule := icahost.NewIBCModule(app.ICAHostKeeper)
 
 	// create static IBC router, add transfer route, then set and seal it
-	// FIXME: Don't be confused by the name!  The port router maps *module names* (not PortIDs) to modules.
+	// FIXME: Don't be confused by the name!  The port router maps *module names*
```

Comment:
is there something to fix here? If not I think `XXX` is a more appropriate comment heading

Reply:
sure (I should have included this ref with my last comment): https://github.com/Agoric/agoric-sdk/wiki/Coding-Style#code-comments